test/core/storage/trie/polkadot_trie/polkadot_trie_test.cpp

```diff
@@ -0,0 +1,334 @@
+    {"0135"_hex2buf, "pen"_buf, Command::GET},
+    {"013579"_hex2buf, "penguin"_buf, Command::GET},
+    {{}, "floof"_buf, Command::PUT},
+    {"f2"_hex2buf, "feather"_buf, Command::PUT},
+
+    {"013579"_hex2buf, "penguin"_buf, Command::REMOVE},
+    {"013579"_hex2buf, boost::none, Command::GET},
+    {"0135"_hex2buf, "pen"_buf, Command::GET},
+    {"013579"_hex2buf, "penguin"_buf, Command::PUT},
+
+    {"0135"_hex2buf, "pen"_buf, Command::REMOVE},
+    {"0135"_hex2buf, boost::none, Command::GET},
+    {"013579"_hex2buf, "penguin"_buf, Command::GET},
+    {"0135"_hex2buf, "pen"_buf, Command::PUT},
+
+    {"013507"_hex2buf, "odd"_buf, Command::REMOVE},
+    {"013579"_hex2buf, "penguin"_buf, Command::GET},
+    {"0135"_hex2buf, "pen"_buf, Command::GET},
+};
+/**
+ * Deletion from a complex branch. BuildSmallTrie must be applied before this
+ * suit
+ */
+std::vector<TrieCommand> DeleteCombineBranch = {
+    {"013546"_hex2buf, "raccoon"_buf, Command::PUT},
+    {"01354677"_hex2buf, "rat"_buf, Command::PUT},
+    {"09d3"_hex2buf, "noot"_buf, Command::REMOVE},
+    {"09d3"_hex2buf, boost::none, Command::GET},
+};
+/**
+ * Deletion from a branch
+ */
+std::vector<TrieCommand> DeleteFromBranch = {
+    {"0615fc"_hex2buf, "noot"_buf, Command::PUT},
+    {"062ba9"_hex2buf, "nootagain"_buf, Command::PUT},
+    {"06afb1"_hex2buf, "odd"_buf, Command::PUT},
+    {"06a3ff"_hex2buf, "stuff"_buf, Command::PUT},
+    {"4321"_hex2buf, "stuffagain"_buf, Command::PUT},
+    {"0615fc"_hex2buf, "noot"_buf, Command::GET},
+    {"062ba9"_hex2buf, "nootagain"_buf, Command::GET},
+    {"0615fc"_hex2buf, "noot"_buf, Command::REMOVE},
+    {"0615fc"_hex2buf, boost::none, Command::GET},
+    {"062ba9"_hex2buf, "nootagain"_buf, Command::GET},
+    {"06afb1"_hex2buf, "odd"_buf, Command::GET},
+    {"06afb1"_hex2buf, "odd"_buf, Command::REMOVE},
+    {"062ba9"_hex2buf, "nootagain"_buf, Command::GET},
+    {"06a3ff"_hex2buf, "stuff"_buf, Command::GET},
+    {"06a3ff"_hex2buf, "stuff"_buf, Command::REMOVE},
+    {"062ba9"_hex2buf, "nootagain"_buf, Command::GET}};
+/**
+ * Deletion on keys with odd length, which might be a problem as a key is
+ * decomposed to 4 bit pieces
+ */
+std::vector<TrieCommand> DeleteOddKeyLengths = {
+    {"43c1"_hex2buf, "noot"_buf, Command::PUT},
+    {"43c1"_hex2buf, "noot"_buf, Command::GET},
+    {"4929"_hex2buf, "nootagain"_buf, Command::PUT},
+    {"4929"_hex2buf, "nootagain"_buf, Command::GET},
+    {"430c"_hex2buf, "odd"_buf, Command::PUT},
+    {"430c"_hex2buf, "odd"_buf, Command::GET},
+    {"4f4d"_hex2buf, "stuff"_buf, Command::PUT},
+    {"4f4d"_hex2buf, "stuff"_buf, Command::GET},
+    {"430c"_hex2buf, "odd"_buf, Command::REMOVE},
+    {"430c"_hex2buf, boost::none, Command::GET},
+    {"f4bc"_hex2buf, "spaghetti"_buf, Command::PUT},
+    {"f4bc"_hex2buf, "spaghetti"_buf, Command::GET},
+    {"4f4d"_hex2buf, "stuff"_buf, Command::GET},
+    {"43c1"_hex2buf, "noot"_buf, Command::GET}};
+
+INSTANTIATE_TEST_CASE_P(GolkadotSuite,
```

Comment:
```suggestion
INSTANTIATE_TEST_CASE_P(PolkadotSuite,
```

Reply:
I think it's referring to the Go implementation.